wiki/Skinning/osu!taiko/en.md

```diff
@@ -1,474 +1,521 @@
-**Notes:**
-
-- Beatmap skinnable status is suspected to be a bug.
-- This element will get stretched to fit screen width.
-- This is the normal state of the scrolling bar.
-
----
-
-![](img/taiko-bar-right-glow.png "taiko-bar-right-glow.png")
-
-| Animatable   | Beatmap Skinnable        |
-|:------------:|:------------------------:|
-| ![No][false] | ![No][false] (see notes) |
-
-**Notes:**
-
-- Beatmap skinnable status is suspected to be a bug.
-- This element will get stretched to fit screen width.
-- This is the kiai state of the scrolling bar.
-- This element overlays `taiko-bar-right`.
-
-## Drumrolls
-
-Drumrolls are the equivalent of the osu!standard sliders.
-
----
-
-![](img/taiko-roll-middle.png "taiko-roll-middle.png")
-
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
-
-**Note:**
-
-- This is the track of the roll where the `sliderscorepoint` are placed on.
-
----
-
-![](img/taiko-roll-end.png "taiko-roll-end.png")
-
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
-
-**Note:**
-
-- This is the end part of a roll.
-
----
-
-![](img/sliderscorepoint.png "sliderscorepoint.png")
-
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![Yes][true]      |
-
-**Note:**
-
-- This is the ticks of the roll.
-
-## Shaker
-
-Shakers are the equivalent of the osu!standard spinner.
-
----
-
-![](img/spinner-warning.png "spinner-warning.png")
-
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![No][false]      |
-
-**Notes:**
-
-- This is an indicator for the spinner.
-
----
-
-![](img/spinner-circle.png "spinner-circle.png")
-
-| Animatable   | Beatmap Skinnable        |
-|:------------:|:------------------------:|
-| ![No][false] | ![No][false] (see notes) |
-
-**Notes:**
-
-- Beatmap skinnable status is suspected to be a bug.
-- For each hit in the spinner happens, the circle rotates anti-clockwise.
-
----
-
-![](img/spinner-approachcircle.png "spinner-approachcircle.png")
-
-| Animatable   | Beatmap Skinnable        |
-|:------------:|:------------------------:|
-| ![No][false] | ![No][false] (see notes) |
-
-**Notes:**
-
-- Beatmap skinnable status is suspected to be a bug.
-- This is the duration indicator of the spinner.
-  - This shrinks over time.
+[true]: /wiki/shared/true.png
+[false]: /wiki/shared/false.png
+
+# osu!taiko
+
+*Main page: [Skinning](/wiki/Skinning).*
+
+*See also: [Skinning osu!taiko Tutorial](/wiki/Skinning_osu!taiko_Tutorial).*
+
+You can override the osu!taiko playfield parts by creating a folder called `taiko` inside your skin folder. If this approach is used, the user will need to explicitly enable this in the [options](/wiki/options) (enable the `Use Taiko skin for Taiko mode` button), otherwise the default skin elements will be used.
+
+## Pippidon
+
+`pippidonclear.png`
+
+![](img/pippidonclear.gif)

```

Comment:
it would be better to animate the pippidonclear.gif like it is programmed in the game.
https://osu.ppy.sh/forum/t/36001 states the order of animation in the following order for clear:
>0 1 2 3 4 5 6 5 6 5 4 3 2 1 0

Additionally state the animation order in the notes